### PR TITLE
Supplement the document

### PR DESCRIPTION
**Pull request by yhwangBonc**

### CODE REVIEW

**Review by neuyilan — CHANGES_REQUESTED** [review]
You've done a great job! But when you edit the document,  please make sure to modify both Chinese and English documents.

**Review by neuyilan — APPROVED** [review]

File: docs/UserGuide/Operation Manual/DDL Data Definition Language.md [hunk 1]
```diff
@@ -309,6 +310,20 @@ IoTDB> unset ttl to root.ln
 
 After unset TTL, all data will be accepted in `root.ln`
 
+## Show TTL
```
Comment (LebronAl) [inline docs/UserGuide/Operation Manual/DDL Data Definition Language.md:313]: same as neuyilan's suggestion~

File: docs/zh/UserGuide/Operation Manual/DDL Data Definition Language.md [hunk 2]
```diff
@@ -41,14 +41,15 @@ Msg: org.apache.iotdb.exception.MetadataErrorException: org.apache.iotdb.excepti
 
 ## 查看存储组
 
-在存储组创建后，我们可以使用[SHOW STORAGE GROUP](../Operation%20Manual/SQL%20Reference.md)语句来查看所有的存储组，SQL语句如下所示：
+在存储组创建后，我们可以使用[SHOW STORAGE GROUP](../Operation%20Manual/SQL%20Reference.md)语句和[SHOW STORAGE GROUP <PrefixPath>](../Operation%20Manual/SQL%20Reference.md)来查看存储组，SQL语句如下所示：
```
Comment (LebronAl) [inline docs/zh/UserGuide/Operation Manual/DDL Data Definition Language.md:44]: image
It seems that the `<PrefixPath>` doesn't display properly.

File: docs/UserGuide/Operation Manual/DDL Data Definition Language.md [hunk 3]
```diff
@@ -41,14 +41,15 @@ Msg: org.apache.iotdb.exception.MetadataErrorException: org.apache.iotdb.excepti
 
 ## Show Storage Group
 
-After the storage group is created, we can use the [SHOW STORAGE GROUP](../Operation%20Manual/SQL%20Reference.md) statement to view all the storage groups. The SQL statement is as follows:
+After the storage group is created, we can use the [SHOW STORAGE GROUP](../Operation%20Manual/SQL%20Reference.md) statement and [SHOW STORAGE GROUP <PrefixPath>](../Operation%20Manual/SQL%20Reference.md) to view the storage groups. The SQL statement is as follows:
```
Comment (LebronAl) [inline docs/UserGuide/Operation Manual/DDL Data Definition Language.md:44]: same as below

**Review by LebronAl — CHANGES_REQUESTED** [review]

File: docs/UserGuide/Operation Manual/DDL Data Definition Language.md (hunk 1 above)
Comment (yhwang-hbl) [reply docs/UserGuide/Operation Manual/DDL Data Definition Language.md:313]: Hi, I changed the Chinese document in the PR submitted earlier,

Comment (yhwang-hbl) [reply docs/UserGuide/Operation Manual/DDL Data Definition Language.md:313]: > same as neuyilan's suggestion~

Hi, I changed the Chinese document in the PR submitted earlier,

File: docs/zh/UserGuide/Operation Manual/DDL Data Definition Language.md (hunk 2 above)
Comment (yhwang-hbl) [reply docs/zh/UserGuide/Operation Manual/DDL Data Definition Language.md:44]: Copy that ,i will improve it

**Review by LebronAl — APPROVED** [review]
LGTM

**Review by HTHou — APPROVED** [review]

File: docs/UserGuide/Operation Manual/DDL Data Definition Language.md [hunk 4]
```diff
@@ -41,14 +41,15 @@ Msg: org.apache.iotdb.exception.MetadataErrorException: org.apache.iotdb.excepti
 
 ## Show Storage Group
 
-After the storage group is created, we can use the [SHOW STORAGE GROUP](../Operation%20Manual/SQL%20Reference.md) statement to view all the storage groups. The SQL statement is as follows:
+After the storage group is created, we can use the [SHOW STORAGE GROUP](../Operation%20Manual/SQL%20Reference.md) statement and [SHOW STORAGE GROUP \<PrefixPath>](../Operation%20Manual/SQL%20Reference.md) to view the storage groups. The SQL statement is as follows:
```
Comment (HTHou) [inline docs/UserGuide/Operation Manual/DDL Data Definition Language.md:44]:
```suggestion
After the storage group is created, we can use the [SHOW STORAGE GROUP](../Operation%20Manual/SQL%20Reference.md) statement and [SHOW STORAGE GROUP \<PrefixPath>](../Operation%20Manual/SQL%20Reference.md) to view the storage groups. The SQL statements are as follows:
```

**Pull request merged**